ZenPacks/zenoss/XenServer/configure.zcml

```diff
@@ -311,6 +311,9 @@
             factory=".impact.IpInterfaceRelationsProvider"
             />
 
+    <!-- Include the zenjobs job definitions. -->
+    <include file="jobs.zcml"/>
```

Comment:
The "include" block is placed inside the _(configure zcml:condition="installed ZenPacks.zenoss.Impact")_ block. Please replace it in the general _configure_ block.